C++arrays (#4494)
Codecademy/docs: content/cpp/concepts/arrays/arrays.md

> In the case above, an array of `chars` was initialized with all the vowels, and then the first element in the array at index `0` was printed out. Then the element at index `0` was modified by being assigned a new value of `'r'`, which was then printed out.

Reviewer: Just to make the paragraph more readable add this 

In the case above, an array of **`chars`** was initialized with all the vowels, and then the first element in the array at index **`0`** was printed out. Subsequently, the element at index **`0`** was modified by assigning it a new value of **`r`**, which was then printed out.

These changes help to refine the flow and clarity of the sentence.

> Arrays in C++ have a fixed size, meaning elements can not be added or removed once the array has been created. Only existing elements may be modified without changing the total size or shape of the structure.

Reviewer: ```suggestion
Arrays in C++ have a fixed size, meaning elements cannot be added or removed once the array has been created. Only existing elements may be modified without changing the total size or shape of the structure.
```

## Uninitialized Elements

> Array elements that are not assigned a value when the array is created are known as uninitialized, and should not be accessed during runtime. For example:

Reviewer: Array elements that are not assigned a value when the array is created are known as uninitialized and should not be accessed during runtime. For example:"

I removed the comma after "uninitialized" to streamline the sentence structure.

```cpp

> int arr[5] = {0, 1, 2}

Reviewer: Added ';' for proper syntax in c++ array.
```suggestion
int arr[5] = {0, 1, 2};
```
